src/website/app/Markdown.js

```diff
@@ -124,6 +106,12 @@ const Image = createStyledComponent('img', {
     width: '100%'
   }
 });
+const ListItem = createStyledComponent('li', ({ theme }) => ({
+  marginBottom: theme.space_stack_sm,
+  '& > *': {
```

Comment:
This is too broad, and will cause issues. For example, I use the following markdown on the home page:

```md
1. **Bolded text**
```

Which results in:

```html
<ol>
  <li><strong>Bolded text</strong></li>
</ol>
```

This selector will force a left margin on that `strong`.

Is there any sort of identifier we could add to the Label to make this selector more targeted. Since our Markdown component requires a bunch of descendant selectors, I think it would even be acceptable to add a static `className` and use that here.

Reply:
fixed